model/project_ref.go

```diff
@@ -2245,54 +2250,78 @@ func (p *ProjectRef) GetGithubProjectConflicts() (GithubProjectConflicts, error)
 }
 
 // ValidateProjectCreation returns a boolean if you should surface the error or not.
-func ValidateProjectCreation(projectId string, config *evergreen.Settings, projectRef *ProjectRef) (bool, error) {
+func ValidateProjectCreation(projectId string, config *evergreen.Settings, projectRef *ProjectRef) (int, error) {
```

Comment:
Can we rename this to include the word `limit`? Like `CheckProjectLimit` or `ValidateProjectCreationLimit`?

Reply:
renamed to `ValidateEnabledProjectsLimit`